README.md

```diff
@@ -0,0 +1,85 @@
+![OpenWrt logo](/logo.svg)
+
+OpenWrt Project is a Linux operating system targeting embedded devices. Instead
+of trying to create a single, static firmware, OpenWrt provides a fully
+writable filesystem with package management. This frees you from the
+application selection and configuration provided by the vendor and allows you
+to customize the device through the use of packages to suit any application.
+For developers, OpenWrt is the framework to build an application without having
+to build a complete firmware around it; for users this means the ability for
+full customization, to use the device in ways never envisioned. 
+
+Latest release: **19.07.3**
+
+Sunshine!
+
+## Development
+
+To build your own firmware you need a Linux, BSD or MacOSX system (case
+sensitive filesystem required). Cygwin is unsupported because of the lack of a
+case sensitive file system.
+
+### Requirements
+
+You need the following tools to compile Openwrt, the package name vary between
+distributions.
+
+### Quickstart
+
+    gcc binutils bzip2 flex python3 perl make find grep diff unzip gawk getopt
+    subversion libz-dev libc-dev
+
+1. Run `./scripts/feeds update -a` to obtain all the latest package definitions
+   defined in feeds.conf / feeds.conf.default
+
+2. Run `./scripts/feeds install -a` to install symlinks for all obtained
+   packages into package/feeds/
+
+3. Run `make menuconfig` to select your preferred configuration for the
+   toolchain, target system & firmware packages.
+
+4. Run `make` to build your firmware. This will download all sources, build the
+   cross-compile toolchain and then cross-compile the Linux kernel & all chosen
+   applications for your target system.
+
+### Related Repositories
+
+The main repository uses multiple sub-repositories to manage packages of
+different categories. If you're looking to develop the web interface port
+packages to OpenWrt, please find the fitting repository below.
+
+* [LuCI Web Interface](https://github.com/openwrt/luci): Modern and modular
+  interface to control the device via a web browser.
+
+* [OpenWrt Packages](https://github.com/openwrt/packages): Ported packages
+  available via the `opkg` packagemanager.
+
+* [OpenWrt Routing](https://github.com/openwrt-routing/packages): Special
+  packages focused on (mesh) routing.
+
+## Support Information
+
+For a list of supported devices see the [OpenWrt Hardware Datanase](https://openwrt.org/supported_devices)
```

Comment:
Datanase->Database

Reply:
Despite that: Generally, for a README, I'd consider to not hide the actual links behind some names, but actually print them.

Reply:
That becomes very messy in my opinion. I think hovering a link to see where it goes it common sense and reading the raw Markdown file show the link and their caption.

Reply:
Will this work with git.openwrt.org?

Reply:
I'd say make it look nice for however visits the web interface and if you look at the raw stuff you get all the underlying information.

Reply:
I'd say it's a README. So, I want to _read_ information here. For fancy links and clickers, create a HTML page. :-)

Reply:
That's what I mean with 21**st** century, Markdown is used to make things visually appealing while still allowing plaintext reading - unlike HTML. I don't thing that hovering a link is to much to ask.